Task2/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+    id 'java'
+}
+
+group 'com.epam.esm'
+version '1.0-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+ext {
+    junitVersion = '5.7.1'
+    springVersion = '5.3.5'
+}
+
+subprojects { project ->
```

Comment:
@Pokemon3108 You don't need to introduce a variable. You can apply everything to subprojects without this